examples/gno.land/r/gnoland/faucet/z2_filetest.gno

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"std"
+
+	"gno.land/p/demo/testutils"
+	"gno.land/r/gnoland/faucet"
+)
+
+// assert render with empty path and 2 controllers
+func main() {
+	var (
+		adminaddr       = std.Address("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5")
+		controlleraddr1 = testutils.TestAddress("controller1")
+		controlleraddr2 = testutils.TestAddress("controller2")
+	)
+	std.TestSetOrigCaller(adminaddr)
+	err := faucet.AdminAddController(controlleraddr1)
+	if err != nil {
+		panic(err)
+	}
+	err = faucet.AdminAddController(controlleraddr2)
+	if err != nil {
+		panic(err)
+	}
+	println(faucet.Render(""))
+}
+
+// Output:
+// # Community Faucet.
+//
+// Status: active.
+// Balance: 200000000ugnot.
+// Total transfers:  (in 0 times).
+//
+// Package address: g17rgsdnfxzza0sdfsdma37sdwxagsz378833ca4
+//
+// Admin: g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5
+//
+//  Controllers:
+//
+//  (unhandled)  (unhandled)
```

Comment:
This is how the controllers are currently printed, I don't think this is the expected result.

Reply:
I created an issue https://github.com/gnolang/gno/issues/645